index.js

```diff
@@ -10,8 +10,10 @@ const execFile = require('child_process').execFile;
 const env = process.env;
 const user = env.LOGNAME || env.USER || env.LNAME || env.USERNAME || '';
 const exclusions = ['--help'];
+const envPaths = require('./env-paths.js');
 
 const configfile = '.v8flags.'+process.versions.v8+'.'+crypto.createHash('md5').update(user).digest('hex')+'.json';
+const configPath = envPaths('js-v8flags').cache;
```

Comment:
The API should be reduced to just return the `.cache` property (and not generate objects with a bunch of properties).  This also removes the need for attribution since it isn't even the same module anymore.